packages/sdks/snippets/angular-16-ssr/src/app/home/homepage.component.ts

```diff
@@ -0,0 +1,42 @@
+import { CommonModule } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {
+  Content,
+  isPreviewing,
+  type BuilderContent,
+} from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-homepage',
+  standalone: true,
+  imports: [Content, CommonModule],
+  template: `
+    <builder-content
+      *ngIf="content || isPreviewing(); else notFound"
+      [model]="MODEL"
+      [content]="content"
+      [apiKey]="API_KEY"
+    ></builder-content>
+    <ng-template #notFound>
+      <div>404</div>
+    </ng-template>
+  `,
+})
+export class HomepageComponent implements OnInit {
+  content: BuilderContent | null = null;
+
+  MODEL = 'homepage';
+  API_KEY = 'ee9f13b4981e489a9a1209887695ef2b';
```

Comment:
why these are all caps? for others pages you've added them all small letters. should we be consistent?